docs/section19/Rule 19-39.md

```diff
@@ -0,0 +1,77 @@
+    - Case 2: Else if the baseline system was modeled with the design supply cfm equal to the maximum OA cfm and the minimum air flow was modeled as the minimum OA flow and some spaces were defined and equal to lab then pass: `if modeled_100_percent_OA == true and modeled_baseline_total_zone_min_OA_CFM >= modeled_proposed_total_zone_min_OA_CFM and are_any_lighting_space_types_defined == true: outcome = "PASS" and raise_message "<insert hvac.id> appears to have been modeled as an 100% outdoor air unit per the requirements of Section G3.1.2.5 Exception #4; however, not all space types associated with the hvac system were defined in the RMD. All defined space types were classed as lab but if any are not lab spaces then recommend manual check to confirm that this unit was modeled correctly per the rules of Section G3.1.2.5 including Exception #4. `  
+    - Case 3: Else if the baseline system was modeled with the design supply cfm equal to the maximum OA cfm and the minimum air flow was modeled as the minimum OA flow and no space types were defined then UNDETERMINED: `if modeled_100_percent_OA == true and modeled_baseline_total_zone_min_OA_CFM >= modeled_proposed_total_zone_min_OA_CFM and are_any_lighting_space_types_defined == false: outcome = "UNDETERMINED" and raise_message "<insert hvac.id> appears to have been modeled as an 100% outdoor air unit; however, no space types associated with the hvac system were defined in the RMD. Recommend manual check to confirm that this unit was modeled correctly per the rules of Section G3.1.2.5 including Exception #4.`    
+    - Case 4: Else, fail: `Else: outcome = "Fail" and raise_message "For <insert hvac.id> either the baseline OA CFM is modeled as less than the proposed or it was not modeled as an 100% percent outdoor air unit. The modeled baseline system serves only laboratory spaces that are likely prohibited from recirculating return air by code or accreditation standards, therefore it is expected that the baseline system would be modeled as 100% outdoor air per the requirements of Section G3.1.2.5 Exception #4 and that the baseline OA cfm would not be less than the proposed."`  
+
+
+**Notes/Questions:**   
+1. I was not sure how exactly to assess whether a unit is 100% OA. I figure it would have to have a design supply cfm that matches the maximum OA cfm and the min supply cfm matching the min OA requirements but I don't think this guarentees 100% OA operation. 
+2. How do we know for sure if the hvac system is prohibited from recirculating return air by code or accreditation standards? In light of this should, we also pass it if OA_CFM_schedule_match == true and its not modeled as 100% OA? I left in the logic for testing OA_CFM_schedule_match == true just in case.
```

Comment:
Perhaps we should add a note to the "PASS" outcome saying that "The proposed system <XXX> severs lab spaces and is modeled as 100% OA system. The corresponding baseline system <xxx> is also modeled with 100% OA. This meets G3.1.2.5 Exception 4 for hvac system that are prohibited from recirculating return air by code or accreditation standards. Recommend a manual check to confirm that recirculating return air is prohibited."

Reply:
Okay, sounds good. Done.